modules/ROOT/pages/policies-policy-categories.adoc

```diff
@@ -0,0 +1,40 @@
+= Policy Categories
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies are categorized by the function they perform. For example, because you are able to resolve issues with the help of logs, the Message Logging policy is categorized as a troubleshooting policy. The following table lists the Default policy name, its category and the function that it performs. +
```

Comment:
The final sentence refers to the default policy, but what about custom policies? Clarify which type of policies this topic includes.
```suggestion
Policies are categorized by the function they perform: compliance, quality of service, security, transformation, and troubleshooting. 

For example, because you are able to resolve issues with the help of logs, the Message Logging policy is categorized as a troubleshooting policy. 

The following table lists the name of the default policy, its category, and the function that it performs. +
```
BUT I suggest that the table be reorganized and introduced this way:
"The following table is organized by policy category, providing the names and functions of each policy within that category."

Reply:
Done! Thanks!